src/features/snackbar/index.js

```diff
@@ -83,7 +75,7 @@ class Snackbar extends Component {
         ) {
             // see if player tried to buy item without enough gold
             this.setState({
-                show: `NOT ENOUGH GOLD FOR: ${notEnoughGold.split('-')[0]}`,
+                show: `NO GOLD FOR: ${notEnoughGold.split('-')[0]}`,
```

Comment:
This really should stay as "NOT ENOUGH", otherwise it's a little misleading as to what's going on (though, "NO" could be appropriate if they player actually had no gold, but then I'm not sure putting the item in really matters for that).

Reply:
While I understand that, it needs to be shortened in order to avoid issues with overflowing text. So it cannot stay as it was, and *some* alternative needs to be found.

Reply:
I would agree, but if we're going to shorten it I think removing the item part would make more sense (I think it should be fairly obvious what item they don't have money for).

Of course, if we want to keep the part about the item we could just make the text smaller, or increase the height of the notification slightly.

Reply:
I can see a solution where it simply says "NOT ENOUGH GOLD", I'll change it to that.